tdrs-backend/tdpservice/settings/local.py

```diff
@@ -53,7 +53,7 @@ class Local(Common):
         from sentry_sdk.integrations.django import DjangoIntegration
         from sentry_sdk.integrations.logging import LoggingIntegration
         sentry_sdk.init(
-            dsn="http://43ebf8abe1434ec6aea2c7b92c465a0e@host.docker.internal:9001/2",
+            dsn="http://e1357afab17cbfe754813aef2ef964ba@192.168.68.59:9001/1",
```

Comment:
TODO: this is for my personal environment. change it for your environment if you are planning to look at Sentry yourself.